models/src/main/java/com/foodie/models/contracts/ResponseHandler.java

```diff
@@ -0,0 +1,32 @@
+package com.foodie.models.contracts;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+public class ResponseHandler {
+    public static ResponseEntity<Object> generateResponse(
+        Exception error, Object message, HttpStatus status) {
+
+        Map<String, Object> map = new HashMap<String, Object>();
+
+        if (error != null && error.getMessage() != null) {
+            map.put("status", status.value());
+            map.put("timestamp", LocalDateTime.now());
+            map.put("error", Arrays.asList(error.getMessage()));
+        }else {
+            if (message instanceof Object) {
```

Comment:
Why are you doing this check, don't you want to check for null instead? Everything in Java is directly or indirectly derived from Object class. In its current state line 25 will never be executed.